teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super()
```

Comment:
- [ ] Kindly initialize Teacher's age, name, and parent_permission as well...
also `super()` mustn't be empty you need to add (age, name, parent_permission) in it.